extensions/data-workspace/src/dataworkspace.d.ts

```diff
@@ -146,4 +152,47 @@ declare module 'dataworkspace' {
 	export interface IProjectActionGroup {
 		actions: IProjectAction[];
 	}
+
+	/**
+	 * Defines project information to be presented in the dashboard container
+	 */
+	export interface IProjectInfo {
+		/**
+		 * name of the table
+		 */
+		tableName: string;
+
+		/**
+		 * column header information
+		 */
+		columnInfo: IDashboardColumnInfo[];
+
+		/**
+		 * data that goes in the table
+		 */
+		tableData: (IDashboardTableData | IDashboardTableDataGroup)[][];
+	}
+
+	/**
+	 * Header information of a column in the table
+	 */
+	export interface IDashboardColumnInfo {
+		displayName: string;
+		width: number;
+		valueType: azdata.DeclarativeDataType;
+	}
+
+	/**
+	 * List of data to be presented on the table in the dashboard container
+	 */
+	export interface IDashboardTableData {
+		value: any;
+	}
+
+	/**
+	 * List of data that should be grouped under one column
+	 */
+	export interface IDashboardTableDataGroup {
```

Comment:
do we really need this?

Reply:
Yes, this is used where more than one component needs to be placed together. For status, I need an icon and a string in one column. For that, having a group is beneficial.
![image](https://user-images.githubusercontent.com/57200045/112560761-178fb380-8d91-11eb-8c74-4539a60f2593.png)

Reply:
well, i wouldn't call that a group, that should be an object {text: string; icon: ...}

Reply:
you don't need to introduce a group for it, it is just one type of data you support.

Reply:
As of now for db projects dashboard, I need icon with string, but for other dashboards, its possible to place other combinations- in that scenario we will have to declare several supported types. Hence thought of adding it generically here. 
But if it still looks incorrect, I can modify it based on your above suggestion.

Reply:
yeah, they can have their own object types, the bottom line is we don't need to introduce a group concept here

Reply:
Sounds good. Will update it accordingly.